stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerDefaultLeadingContent.kt

```diff
@@ -40,14 +42,12 @@ public class MessageComposerDefaultLeadingContent : FrameLayout, MessageComposer
      * Initial UI rendering and setting up callbacks.
      */
     private fun init() {
-        binding = StreamUiMessageComposerDefaultLeadingContentBinding.inflate(streamThemeInflater, this)
-        binding.attachmentsButton.setOnClickListener { onAttachmentsButtonClicked() }
-        binding.commandsButton.setOnClickListener { onCommandsButtonClicked() }
+        binding.attachmentsButton.setOnClickListener { onAttachmentsButtonClick() }
+        binding.commandsButton.setOnClickListener { onCommandsButtonClick() }
     }
 
     /**
      * Re-rendering the UI according to the new state.
      */
-    override fun renderState(state: MessageComposerState) {
-    }
+    override fun renderState(state: MessageComposerState): Unit = Unit
```

Comment:
Should we maybe disable commands/integrations here in case there is input filled and/or we have attachments, are editing etc?

Reply:
Yes, I will do this in a separate PR